tasks/README.md

```diff
@@ -238,4 +238,7 @@ Name | Summary | Category
 `task301_record_question_generation` | Given a passage, generate a fill-in-the-gap question based on it | Question Generation
 `task302_record_classification` | Given a passage and a question, classify the answer to the question based on the options | Classification
 `task303_record_incorrect_answer_generation` | Given a passage and a question, write an incorrect answer for the question | Incorrect Answer Generation
-`task339_record_answer_generation` | Given a passage and a question, answer the question based on the passage | Answer Generation
\ No newline at end of file
+`task339_record_answer_generation` | Given a passage and a question, answer the question based on the passage | Answer Generation
+`task360_SPOLIN_YesAnd_response_generation` | Given a prompt, generate the "yes, ands" response | Response Generation
+`task361_SPOLIN_YesAnd_prompt_response_classification` | Given a prompt and a response, classify whether the response is "yes, ands" type | Classification
```

Comment:
we gotta lowercase these names and merge with the main branch.

Reply:
Renamed the files.